tests/tests.rs

```diff
@@ -1,8 +1,10 @@
-use stellar_strkey::*;
+#![no_std]
```

Comment:
Tests always run with std imported anyway, because the test framework itself uses std, so I don't think we need the changes in this file.

Is there a reason using alloc explicitly here benefits?

Reply:
We could add a crate to the repo that imports strkey and gets built as part of testing, that provides it's own panic handler, because the only way it'll build is if std isn't imported because there can't be two panic handler and std provides one. 

https://doc.rust-lang.org/nomicon/panic-handler.html

Reply:
On the same note that's a lot of test machinery for checking something that is unlikely to be the case since the no_std attribute is in use.

The risk is that it depends on a crate that pulls in std.

I wonder if something like https://github.com/cackle-rs/cackle could detect it.

Reply:
Hi @leighmcculloch, I have integrated version 0.0.11 into Ledger, and it's working very well. Currently, I want to make rs-stellar-xdr support no std, so I will work on the tests here and consider adding no alloc support in my free time.

Reply:
There's nothing riding on no alloc, so if it's not of value to you, I suggest holding off as there's nothing htat I know of needing it right now.